website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,353 @@
+---
+description: Deploy Appsmith on a Kubernetes cluster
+sidebar_position: 3
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Kubernetes
+
+[Kubernetes[K8s]](https://kubernetes.io/docs/concepts/overview/) is an open source platform for managing, scaling, and automating the deployment of [containerized applications](https://en.wikipedia.org/wiki/Containerization). It's one of the most popular container orchestration tools. The guide helps you to install [Appsmith](https://github.com/appsmithorg/appsmith) on a [Kubernetes](https://github.com/appsmithorg/appsmith/blob/release/deploy/helm/kubernetes.io) cluster using [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+Before installing Appsmith, a few prerequisites need to be installed and configured.
+1. Install the [Helm Package Manager](https://helm.sh/docs/intro/install/)- Helm is a package manager for Kubernetes. It allows you to install and manage applications on Kubernetes. To install Helm, follow the instructions for your platform from the official [Helm documentation](https://helm.sh/docs/intro/install/).
+2. Install [`kubectl`](https://kubernetes.io/vi/docs/tasks/tools/install-kubectl/) - `kubectl` is the command-line interface for Kubernetes. It allows you to run commands against Kubernetes clusters to manage applications and other resources. To install kubectl, follow the instructions for your platform from the official [Kubernetes documentation](https://kubernetes.io/vi/docs/tasks/tools/install-kubectl/).
+
+### Configure Kubernetes cluster
```

Comment:
this looks good. But I'm just a little concerned about whether users will consider this as a prerequisite as it got disconnected from the points above.

Reply:
Hmm. I'll add guideline for that.

Reply:
Done.